dev/generate-tests.py

```diff
@@ -959,6 +960,7 @@ def gencudakerneltests(specdict):
 
                 f.write(
                     "import cupy\n"
+                    "import cupy.testing as cpt\n"
```

Comment:
```suggestion
                    "import cupy.testing as cpt\n"
                    "import numpy as np\n"
```

Reply:
ok, it looks like the errors are caused by a missing kernel - `awkward_ListOffsetArray_reduce_nonlocal_preparenext_64`